src/main.rs

```diff
@@ -10,13 +10,14 @@ mod icon;
 // When compiling natively:
 #[cfg(not(target_arch = "wasm32"))]
 fn main() {
+    
```

Comment:
Please remove the newline to appease `rustfmt`.

It can be done automatically using the `cargo fmt` command